packages/proxyIdentity/contracts/IdentityManager.sol

```diff
@@ -1,12 +1,40 @@
-pragma solidity 0.8.0;
+pragma solidity 0.4.26;
 
+import "@optionality.io/clone-factory/contracts/CloneFactory.sol";
+import "openzeppelin-solidity/contracts/introspection/ERC165Checker.sol";
+import "./OfferableIdentity.sol";
+
+
+contract IdentityManager is CloneFactory {
+  address libraryAddress;
```

Comment:
This is the address of the implementation of `OfferableIdentity` that is being cloned, right? Maybe a name like `IdentityImplementation` might be a little more specific?

Reply:
It is address of contract containing logic of all clones. Clones (which has only state) will delegate all calls to it. So it is used as library